src/accordion/accordion.directive.ts

```diff
@@ -32,20 +34,24 @@ let nextId = 0;
  *
  * @since 14.1.0
  */
-@Directive({
+@Component({
 	selector: '[ngbAccordionBody]',
 	standalone: true,
+	template: `
+		<ng-container #container />
+		<ng-content />
+	`,
 	host: {
 		class: 'accordion-body',
 	},
 })
 export class NgbAccordionBody implements AfterContentChecked, OnDestroy {
-	private _vcr = inject(ViewContainerRef);
-	private _element = inject(ElementRef<HTMLElement>).nativeElement;
 	private _item = inject(NgbAccordionItem);
-
 	private _viewRef: EmbeddedViewRef<any> | null = null;
 
+	public readonly elementRef = inject(ElementRef);
```

Comment:
just don't forget to put the comment with `@since x.x.x` annotation when doing a release, so it would be documented and visible in API documentation